### PR TITLE
Migration: add forced reset template

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Add template for a Forced Reset Password email.

### CODE REVIEW

File: migrations/versions/0419_add_forced_pass_template.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+revision = "0419_add_forced_pass_template"
+down_revision = "0327_add_password_expired"
+
+
+user_id = "6af522d0-2915-4e52-83a3-3690455a5fe6"
+service_id = "d6aa2c68-a2d9-4437-ab19-3ae8eb202553"
+
+
+def upgrade():
+    password = hashpw(str(uuid.uuid4()))
```
Comment (sastels) [inline migrations/versions/0419_add_forced_pass_template.py:27]: What is this used for?

Comment (jzbahrai) [reply migrations/versions/0419_add_forced_pass_template.py]: this was a mistake

Comment (jzbahrai) [reply migrations/versions/0419_add_forced_pass_template.py]: i added the template history table as well if you can review. 🙇🏽‍♀️

File: migrations/versions/0419_add_forced_pass_template.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        Pour réinitialiser votre mot de passe, veuillez cliquer sur le lien suivant :
+        [Réinitialisation de votre mot de passe](((url))?lang=fr)
+        Ce lien est unique. Ne le transmettez à personne. 
+        Si vous navez pas demandé ce courriel, veuillez [nous contacter](https://notification.canada.ca/contact?lang=fr).
+        """
+
+    op.execute(
+        template_history_insert.format(
+            "e9a65a6b-497b-42f2-8f43-1736e43e13b3",
+            "Notify email verification code",
```
Comment (jimleroyer) [inline migrations/versions/0419_add_forced_pass_template.py:56]: Should it be the same name as the one in the templates table?
```suggestion
            "Notify forced-password reset email",
```

Comment (jzbahrai) [reply migrations/versions/0419_add_forced_pass_template.py:56]: will change

**Review by jimleroyer — APPROVED** [review]
LGTM

**Pull request merged**